File: week 6/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

if n == 1:
    print("#")

if n == 2:
    print(" #")
    print("##")

if n == 3:
    print("  #")
    print(" ##")
    print("###")

if n == 4:
    print("   #")
    print("  ##")
    print(" ###")
    print("####")

if n == 5:
    print("    #")
    print("   ##")
    print("  ###")
    print(" ####")
    print("#####")

if n == 6:
    print("     #")
    print("    ##")
    print("   ###")
    print("  ####")
    print(" #####")
    print("######")

if n == 7:
    print("      #")
    print("     ##")
    print("    ###")
    print("   ####")
    print("  #####")
    print(" ######")
    print("#######")

if n == 8:
    print("       #")
    print("      ##")
    print("     ###")
    print("    ####")
    print("   #####")
    print("  ######")
    print(" #######")
    print("########")

File: week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int a;
    do
    {
        // Asking the user for the pyramid height 'a'
        a = get_long("Height: ");

        // Number of brics between 1 - 8
        if (a > 0 && a <= 8)
        {
            // Print the bric #
            switch (a)
            {
                case 8:
                    printf("       #  #\n");

                    printf("      ##  ##\n");

                    printf("     ###  ###\n");

                    printf("    ####  ####\n");

                    printf("   #####  #####\n");

                    printf("  ######  ######\n");

                    printf(" #######  #######\n");

                    printf("########  ########\n");
                    break;

                case 7:
                    printf("      #  #\n");

                    printf("     ##  ##\n");

                    printf("    ###  ###\n");

                    printf("   ####  ####\n");

                    printf("  #####  #####\n");

                    printf(" ######  ######\n");

                    printf("#######  #######\n");
                    break;

                case 6:
                    printf("     #  #\n");

                    printf("    ##  ##\n");

                    printf("   ###  ###\n");

                    printf("  ####  ####\n");

                    printf(" #####  #####\n");

                    printf("######  ######\n");
                    break;

                case 5:
                    printf("    #  #\n");

                    printf("   ##  ##\n");

                    printf("  ###  ###\n");

                    printf(" ####  ####\n");

                    printf("#####  #####\n");
                    break;

                case 4:
                    printf("   #  #\n");

                    printf("  ##  ##\n");

                    printf(" ###  ###\n");

                    printf("####  ####\n");
                    break;

                case 3:
                    printf("  #  #\n");

                    printf(" ##  ##\n");

                    printf("###  ###\n");
                    break;

                case 2:
                    printf(" #  #\n");

                    printf("##  ##\n");
                    break;

                case 1:
                    printf("#  #\n");
                    break;
            }
        }
    }
    while (a > 8 || a <= 0);
}